tests/unit/utils/test_utils.py

```diff
@@ -878,3 +878,31 @@ def test_parse_duration_hypothesis(duration):
 ])
 def test_mimetype_extension(mimetype, extension):
     assert utils.mimetype_extension(mimetype) == extension
+
+
+class TestCleanupFileContext:
+
+    def test_no_file(self, tmpdir):
```

Comment:
Please use `tmp_path` instead, as it's the more modern replacement using the `pathlib.Path` API instead of pytests own one - see https://docs.pytest.org/en/stable/tmpdir.html and #176.

Reply:
Done